paulmikulskis/redditat
# added Sentry on chrome extension (only when production == NODE_ENV)

test:
1. get fetch origin and git checkout john/15-add-sentry-on-chrome-extension
2. load the extension: spamcntrl-extension/extension/chrome
3. change the process.env.NODE_ENV == "production" to development
4. throw an error on CDashboardPage:
`
useEffect(() => {
 throw new Error("Sample Error")
}, [])
`
you should see it added on [sentry](https://sentry.yungstentech.com/organizations/yungsten-tech/issues/?project=4#welcome): https://prnt.sc/_0PFfOCHo3ft
